src/mono/nuget/Microsoft.NET.Sdk.WebAssembly.Pack/build/Microsoft.NET.Sdk.WebAssembly.Browser.targets

```diff
@@ -11,6 +11,19 @@ Copyright (c) .NET Foundation. All rights reserved.
 -->
 <Project ToolsVersion="14.0">
 
+  <PropertyGroup>
+    <_UseBlazorDevServer>$(RunArguments.Contains('blazor-devserver.dll').ToString().ToLower())</_UseBlazorDevServer>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(_WebAssemblyUserRunParameters)' == '' and '$(_UseBlazorDevServer)' == 'false'">
+    <RunCommand Condition="'$(DOTNET_HOST_PATH)' != '' and Exists($(DOTNET_HOST_PATH))">$(DOTNET_HOST_PATH)</RunCommand>
+    <RunCommand Condition="'$(RunCommand)' == ''">dotnet</RunCommand>
+
+    <WasmAppHostDir>$([MSBuild]::NormalizeDirectory($(MSBuildThisFileDirectory), '..', 'WasmAppHost'))</WasmAppHostDir>
+    <_RuntimeConfigJsonPath>$([MSBuild]::NormalizePath($(OutputPath), '$(AssemblyName).runtimeconfig.json'))</_RuntimeConfigJsonPath>
+    <RunArguments>exec &quot;$([MSBuild]::NormalizePath($(WasmAppHostDir), 'WasmAppHost.dll'))&quot; --runtime-config &quot;$(_RuntimeConfigJsonPath)&quot; $(WasmHostArguments)</RunArguments>
+    <RunWorkingDirectory>$(OutputPath)</RunWorkingDirectory>
```

Comment:
This can be tricky to get correct. How did you test it? @ilonatommy has tested `dotnet run` wasmapphost before too.

Reply:
I played around with simple echo and printing out various variables in various places. Later I added `Debugger.Break()` at the beginning of wasmapphost to be able to debug arguments etc. Is that what your question was about?

Reply:
No, I meant, running the app with `dotnet run`, and with `--no-build`, and running it from the project directory, and outside.

Reply:
Scenarios:
- Create a new WebAssembly Browser App on VS and run, change WasmAppDir and run again.
- Create a new WebAssembly Console App on VS and run, change WasmAppDir and run again.
- Create a new wasmbrowser app using command line and run;
- Create a new wasmconsole app using command line and run;
- Create a new wasmbrowser app using command line and dotnet build and dotnet run --no-build;
- Create a new wasmbrowser app using command line and dotnet build and dotnet run --debug;

Reply:
The `WasmAppDir` not relevant for WasmSDK. Tested the rest.